src/coreclr-debug/activate.ts

```diff
@@ -23,32 +23,39 @@ export async function activate(thisExtension : vscode.Extension<any>, context: v
     _logger = logger;
 
     if (!CoreClrDebugUtil.existsSync(_debugUtil.debugAdapterDir())) {
-        let platformInformation: PlatformInformation;
-        
-        try {
-            platformInformation = await PlatformInformation.GetCurrent();
-        }
-        catch (err) {
-            // Somehow we couldn't figure out the platform we are on
+        let isInvalidArchitecture: boolean = await checkForInvalidArchitecture(logger, channel);
```

Comment:
I think this is going to be really annoying for anyone who is using the extension for editing C#. I think we want to only do this when we first try to install, and when the user tries to debug.

Reply:
Never mind. I guess this is what we always have done.